### PR TITLE
Added 12 icons (some for MIUI), Updated 1, Linked 1

### PR DESCRIPTION
**Pull request by mluizvitor**
## Type of change

:x: Bug fix (non-breaking change which fixes an issue)
:white_check_mark: Icon addition (non-breaking change that adds/modifies Lawnicons's icons)
:x: General change (non-breaking change that doesn't fit the above categories like copyediting)

### Icons added
* AliExpress (`com.alibaba.aliexpresshd`)
* AnyDesk (`com.anydesk.anydeskandroid`)
* Urbano Norte (`br.com.urbanonorte.passenger.drivermachine`)
* 99 (`com.taxis99`)
* Miui Compass (`com.miui.compass`)
* Miui Notes (`com.miui.notes`)
* Miui Scanner (`com.xiaomi.scanner`)
* QR Reader (`com.teacapps.barcodescanner`)
* Miui Recorder (`com.android.soundrecorder`)
* Miui Screen Recorder (`com.miui.screenrecorder`)
* Miui Weather (`com.miui.weather2`)
* Miui Themes (`com.android.thememanager`)

### Icons Updated
* Repainter (`dev.kdrag0n.dyntheme`)

### Linked Icons
* Miui FM Radio (`com.miui.fm`)

### CODE REVIEW

**Review by Chefski — CHANGES_REQUESTED** [review]
Hey! Thanks for contributing! I've got some changes to some icons:

Compass, download, notes, QR reader, repainter could be sized up to fit the 160x160px content area
Screenshot 2022-09-25 at 21 34 58

`screen_recorder.svg`is also off centre. Mind repositioning it?

`weather.svg` is off centre and a bit bigger than the 160x160px content area. Mind fixing that too? Thanks

File: app/src/main/res/xml/grayscale_icon_map.xml [hunk 1]
```diff
@@ -555,6 +563,7 @@
     <icon drawable="@drawable/rabobank" package="nl.rabomobiel" name="rabobank" />
     <icon drawable="@drawable/raindrop" package="io.raindrop.raindropio" name="Raindrop" />
     <icon drawable="@drawable/recorder" package="com.android.bbksoundrecorder" name="Recorder" />
+    <icon drawable="@drawable/recorder" package="com.android.soundrecorder" name="Recorder"/>
```
Comment (Chefski) [inline app/src/main/res/xml/grayscale_icon_map.xml:566]:
```suggestion
    <icon drawable="@drawable/recorder" package="com.android.soundrecorder" name="Recorder" />
```

File: app/src/main/res/xml/grayscale_icon_map.xml [hunk 2]
```diff
@@ -660,6 +671,7 @@
     <icon drawable="@drawable/the_battle_cats" package="jp.co.ponos.battlecatsen" name="The Battle Cats" />
     <icon drawable="@drawable/the_economist" package="com.economist.lamarr" name="Economist" />
     <icon drawable="@drawable/theathletic" package="com.theathletic" name="The Athletic" />
+    <icon drawable="@drawable/themes" package="com.android.thememanager" name="Themes"/>
```
Comment (Chefski) [inline app/src/main/res/xml/grayscale_icon_map.xml:674]:
```suggestion
    <icon drawable="@drawable/themes" package="com.android.thememanager" name="Themes" />
```

File: app/src/main/res/xml/grayscale_icon_map.xml [hunk 3]
```diff
@@ -683,6 +695,7 @@
     <icon drawable="@drawable/twitter" package="com.twitter.android.lite" name="Twitter" />
     <icon drawable="@drawable/twitter" package="com.twitter.android" name="Twitter" />
     <icon drawable="@drawable/unidays" package="com.myunidays" name="Unidays" />
+    <icon drawable="@drawable/urbano_norte" package="br.com.urbanonorte.passenger.drivermachine" name="Urbano Norte"/>
```
Comment (Chefski) [inline app/src/main/res/xml/grayscale_icon_map.xml:698]:
```suggestion
    <icon drawable="@drawable/urbano_norte" package="br.com.urbanonorte.passenger.drivermachine" name="Urbano Norte" />
```

File: app/src/main/res/xml/grayscale_icon_map.xml [hunk 4]
```diff
@@ -697,6 +710,7 @@
     <icon drawable="@drawable/waze" package="com.waze" name="Waze" />
     <icon drawable="@drawable/wear_os" package="com.google.android.wearable.app" name="Wear OS" />
     <icon drawable="@drawable/wear" package="com.samsung.android.app.watchmanager" name="Samsung Wear" />
+    <icon drawable="@drawable/weather" package="com.miui.weather2" name="Weather"/>
```
Comment (Chefski) [inline app/src/main/res/xml/grayscale_icon_map.xml:713]:
```suggestion
    <icon drawable="@drawable/weather" package="com.miui.weather2" name="Weather" />
```

File: app/src/main/res/xml/grayscale_icon_map.xml [hunk 5]
```diff
@@ -464,8 +469,10 @@
     <icon drawable="@drawable/newpipe" package="org.schabi.newpipelegacy" name="NewPipe" />
     <icon drawable="@drawable/nextcloud" package="com.nextcloud.android.beta" name="Nextcloud Dev" />
     <icon drawable="@drawable/nextcloud" package="com.nextcloud.client" name="Nextcloud" />
+    <icon drawable="@drawable/ninenine" package="com.taxis99" name="99" />
     <icon drawable="@drawable/notally" package="com.omgodse.notally" name="Notally" />
     <icon drawable="@drawable/notenapp" package="com.notenapp" name="Notenapp" />
+    <icon drawable="@drawable/notes" package="com.miui.notes" name="Notes"/>
```
Comment (Chefski) [inline app/src/main/res/xml/grayscale_icon_map.xml:475]:
```suggestion
    <icon drawable="@drawable/notes" package="com.miui.notes" name="Notes" />
```

File: app/src/main/res/xml/grayscale_icon_map.xml [hunk 6]
```diff
@@ -151,6 +153,7 @@
     <icon drawable="@drawable/coinbase" package="com.coinbase.android" name="Coinbase" />
     <icon drawable="@drawable/coins_ph" package="asia.coins.mobile" name="Coins.ph" />
     <icon drawable="@drawable/coinvero" package="com.candlestick_app" name="Coinvero" />
+    <icon drawable="@drawable/compass" package="com.miui.compass" name="Compass"/>
```
Comment (Chefski) [inline app/src/main/res/xml/grayscale_icon_map.xml:156]:
```suggestion
    <icon drawable="@drawable/compass" package="com.miui.compass" name="Compass" />
```

File: app/src/main/res/xml/grayscale_icon_map.xml [hunk 7]
```diff
@@ -18,6 +19,7 @@
     <icon drawable="@drawable/ankidroid" package="com.ichi2.anki" name="AnkiDroid" />
     <icon drawable="@drawable/antutu" package="com.antutu.ABenchMark" name="AnTuTu Benchmark" />
     <icon drawable="@drawable/antutu" package="com.antutu.benchmark.full" name="AnTuTu 3DBench" />
+    <icon drawable="@drawable/anydesk" package="com.anydesk.anydeskandroid" name="AnyDesk"/>
```
Comment (Chefski) [inline app/src/main/res/xml/grayscale_icon_map.xml:22]:
```suggestion
    <icon drawable="@drawable/anydesk" package="com.anydesk.anydeskandroid" name="AnyDesk" />
```

**Review by Chefski — CHANGES_REQUESTED** [review]

**Comment by mluizvitor** [comment]
Hello @Chefski. Thanks for the feedback. `screen_recorder.svg` and `weather.svg` where made off center for visual alignment. I'll center them and resolve the other issues.

**Comment by mluizvitor** [comment]
Just updated the icons. I can't ignore the fact that some icon shapes are bigger or smaller than others as spotted on the issue https://github.com/LawnchairLauncher/lawnicons/issues/471

Previously
image

After Update
image

**Review by Chefski — APPROVED** [review]
Thanks for fixing :)

**Comment by Chefski** [comment]
@Goooler

**Comment by SuperDragonXD** [comment]
Regarding the "some icon shapes are bigger or smaller than others", it is due to the scaling of the icons by the launcher. Because if you look at the icons on the actual Lawnicons app, it looks fine and aren't scaled.

**Pull request merged**